### DISCUSSION TITLE
Ctrl/CMD + Z in a accidentally moved box in diagram

### DISCUSSION THREAD

**Discussion by leopascual** [postmanlabs/postman-flows#86] [Ideas]
There are no states in the flow, it would be nice to redo something it was moved by accident to the position in the diagram. 

AWESOME Feature **FLOW** BTW,  love it.

**Comment by samay-v** [comment]
Thanks @leopascual! 😄 
We're tracking the ability to undo and redo here https://github.com/postmanlabs/postman-flows/issues/29. You should be able to Ctrl/CMD + Z to undo soon
